README.md

```diff
@@ -89,6 +89,41 @@ Create a PR for your changes and once it is merged then Colmena will manage the
 machine for you and you will see it listed as a client in the Nomad dashboard
 at <https://nomad-server-01.holochain.org:4646/ui/clients>.
 
+### Using Published Docker Image
+
+> [!WARNING]\
+> The wind-tunnel-runner Docker image requires extensive permissions on the
+> host machine that are effectively root access. It should only be run on a
+> dedicated machine as the sole Docker container.
+
+First, install [Docker](https://docs.docker.com/get-started/get-docker/) on your
+machine, and start the Docker service.
+
+Next, pull the latest Docker image for the wind-tunnel-runner:
+
+```bash
+docker image pull ghcr.io/holochain/wind-tunnel-runner:latest
+```
+
+Finally, run the Docker container. Replace `<MY_HOSTNAME>` with a hostname for
+your machine, and run the command below:
+
+The hostname should be unique and recognisable so that you know which node
+belongs to you, something like `nomad-client-<user>` is common practice with a
+base-one index after it, if you have multiple machines,
+i.e. `nomad-client-<user>-<index>`.
+
+```bash
+docker run --hosname <MY_HOSTNAME> --cgroupns=host --privileged -d -t --rm ghcr.io/holochain/wind-tunnel-runner:latest
```

Comment:
I realized one more question for the docker setup -- currently the holochain conductor config in wind tunnel scenarios uses the default settings for network.advanced.tx5Transport.ephemeral_udp_port_min and network.advanced.tx5Transport.ephemeral_udp_port_max. This means the conductor will pick any open ephemeral port for webrtc connections.

Docker falls over if you try to bind all possible ephemeral udp ports to the host networking. We have 2 options:

1. We could keep this config as-is, and run the docker container with `--net=host`
2. We could modify the holochain config in wind-tunnel scenarios to set a smaller port range (say 50 ports), and then run the docker container with `--p <min>-<max>:<min>-<max>` to publish only that port range.

I would probably do 1 for now for simplicity.